src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -81,10 +92,13 @@ class SlickDataAccess(databaseConfig: Config, val dataAccess: DataAccessComponen
   import dataAccess.driver.api._
 
   // NOTE: if you want to refactor database is inner-class type: this.dataAccess.driver.backend.DatabaseFactory
-  val database = Database.forConfig("", databaseConfig.withUniqueSchema)
+  private val configWithUniqueSchema = databaseConfig.withUniqueSchema
+  val database = Database.forConfig("", configWithUniqueSchema)
 
   // Possibly create the database
   {
+    import SlickDataAccess._
+    log.info(s"Running with database ${configWithUniqueSchema.urlKey} = ${configWithUniqueSchema.urlValue}")
```

Comment:
I know @coreone had concerns about passwords being logged, not sure if that would happen here.